<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="questionTextLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Тут будет вопрос...</string>
    </property>
   </widget>
   <widget class="QLabel" name="answerText">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>640</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Тут ответ (диагноз)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="yesPushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Yes</string>
    </property>
   </widget>
   <widget class="QPushButton" name="noPushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Задано вопросов:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>1041</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="questionsDoneLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Осталось вопросов:</string>
    </property>
   </widget>
   <widget class="QLabel" name="questionsLeft">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>440</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>16</width>
      <height>381</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>640</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ответ: </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="excludedDiagnosisTableWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Исключенные диагнозы</string>
    </property>
   </widget>
   <widget class="QLabel" name="countExcludedDiagnosesLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Диагнозы и их вероятности </string>
    </property>
   </widget>
   <widget class="QPushButton" name="startpushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>640</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="countAllQuestionLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>420</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTableWidget" name="leftDiagnosisTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>748</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
